### PR TITLE
Update step sequencer tutorial

### PR DESCRIPTION
**Pull request by Rumyra**
Most of the edits here are within the code examples to pull them inline with including a `time` param for scheduling.

See issue https://github.com/mdn/content/issues/2310 and demo fix PR https://github.com/mdn/webaudio-examples/pull/37

Also updating copy to reflect and fixing flaws

### CODE REVIEW

File: files/en-us/web/api/web_audio_api/advanced_techniques/index.html [hunk 1]
```diff
@@ -429,12 +427,12 @@ <h3 id="The_final_playSample_function">The final playSample() function</h3>
 
 <p>We'll then add a line to update the <code>playbackRate</code> property to our <code>playSample()</code> function. The final version looks like this:</p>
 
-<pre class="brush: js">function playSample(audioContext, audioBuffer) {
+<pre class="brush: js">function playSample(audioContext, audioBuffer, time) {
     const sampleSource = audioContext.createBufferSource();
     sampleSource.buffer = audioBuffer;
-    sampleSource.playbackRate.setValueAtTime(playbackRate, audioCtx.currentTime);
+    sampleSource.playbackRate = playbackRate;
```
Comment (sbungartz) [inline files/en-us/web/api/web_audio_api/advanced_techniques/index.html:433]:
```suggestion
    sampleSource.playbackRate.value = playbackRate;
```

File: files/en-us/web/api/web_audio_api/advanced_techniques/index.html [hunk 2]
```diff
@@ -493,22 +489,24 @@ <h2 id="Playing_the_audio_in_time">Playing the audio in time</h2>
...
     }
-    if (pads[2].querySelectorAll('button')[currentNote].getAttribute('aria-checked') === 'true') {
-        playNoise()
+    if (pads[2].querySelectorAll('button')[beatNumber].getAttribute('aria-checked') === 'true') {
+        playNoise(time)
     }
-    if (pads[3].querySelectorAll('button')[currentNote].getAttribute('aria-checked') === 'true') {
-        playSourceNode(audioCtx, sample);
+    if (pads[3].querySelectorAll('button')[beatNumber].getAttribute('aria-checked') === 'true') {
+        playSourceNode(audioCtx, dtmf, time);
```
Comment (sbungartz) [inline files/en-us/web/api/web_audio_api/advanced_techniques/index.html:502]: This seems to have been renamed at some point, and forgotten here, if I'm not mistaken:
```suggestion
        playSample(audioCtx, dtmf, time);
```

**Review by sbungartz — COMMENTED** [review]
Hi @Rumyra, thank you for also updating the Article! I noticed two small mistakes (see suggestions), but it looks great to me :+1:

**Comment by Rumyra** [comment]
Ha - good spots @sbungartz ! I think those have always been there 😬 - all committed 👍

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**